geps/gep-1709.md

```diff
@@ -97,25 +92,42 @@ conformance suite as a library.
 
 ### Profiles
 
-Initially there will be two profiles:
-
-- `Layer4`
-- `Layer7`
-
-> **NOTE**: these are simply the initial profiles we're going to start with,
-> it's plausible for there to be more in the future.
-
-These named profiles are effectively categories which represent the high level
-grouping of tests. When conformance is reported using one of these profiles
-extra features can be covered according to support levels:
+"Profiles" are effectively categories which represent the high level grouping of
+tests related to some feature (or feature set) of Gateway API. When conformance
+is reported using one of these profiles extra features can be covered according
+to support levels:
 
+- `core`
 - `extended`
 
 > **NOTE**: `implementation-specific` doesn't really have much in the way of
 > tests today, but it is something users want to be able to display. We leave
 > door open for it later and mention it in the [alternatives
 > considered](#alternatives-considered) section below.
 
+We will start with the following named profiles:
+
+- `HTTP`
+- `TLS`
```

Comment:
does this cover passthrough or termination ?

Reply:
This is a good question, it's probably safest to just name these profiles `HTTPRoute`, `TLSRoute`, etc, to avoid any confusion with overall protocol support.

Reply:
@sunjayBhatia any thoughts here?

Reply:
thinking about the `TLS` category name a little, it is a bit broad, you can do HTTP over TLS, gRPC over TLS, some other TCP protocol etc., and then you get the terminate/passthrough configuration option

I think as-written the `TLS` category is intended to cover [this existing test](https://github.com/kubernetes-sigs/gateway-api/blob/40a86dcad8f2dd307435b82317483248d484f853/conformance/tests/tlsroute-simple-same-namespace.go) and any other related additional tests. Given our stance on listener protocol/route type (https://gateway-api.sigs.k8s.io/guides/tls/#clientserver-and-tls), TLSRoute is only really used for passthrough TLS, so conceptually I think we can express that.

IMO we could rename `TLS` here to `TLSPassthrough`

Reply:
Let's go for that, again we're still in provisional/prototyping so we've got lots of runway to try this out and see how it feels.

```suggestion
- `TLSPassthrough`
```

Please anyone unresolve if you feel that there's more action to take here :+1:

Reply:
`ReferenceGrant` is actually one that's been troubling me. We have a note about how `GatewayClass` and `Gateway` will be automatically included as they are referent in some featuresets, see here:

https://github.com/kubernetes-sigs/gateway-api/pull/1783/files#diff-7764ccb47f68a80348b14a2c6df6c6cadd80db02de7817dbcb8039a611bf1328R126

However I only put `GatewayClass` and `Gateway` there because those are easy and obvious, `ReferenceGrant` _might_ fit there but it's a little more subtle.

Since we'll need to make sure we account for this before we go GA, but I would prefer not to get this PR hung up for it I've added an explicit TODO item saying that we'll need to resolve this before `implementable`:

https://github.com/kubernetes-sigs/gateway-api/pull/1783/commits/d7bd41733e5f6ce40bbd45cb7f09ddac0863a71c